src/css/styles/_general.scss

```diff
@@ -22,3 +23,61 @@
 .row {
   margin: 0;
 }
+
+.top {
+  position: relative;
+  width: 100%;
+  min-height: 500px;
+  overflow: hidden;
+  background: url("../img/banner-bg.jpg") no-repeat 75%;
+  background-size: cover;
+}
+
+.banner {
+  padding-top: 60px;
+}
+
+.repair1,
+.repair2,
+.production1,
+.production2 {
+  margin-bottom: 6rem;
+}
+
+.experience__bg {
+  width: 100%;
+  overflow: hidden;
+  background: url("../img/experience-bg.jpg") no-repeat 15%;
+  background-size: cover;
+}
+
+.experience__logo {
+  width: 243.5px;
+  margin: 0 auto;
+}
+
+.experience__logo-img {
+  max-width: 243.5px;
+}
```

Comment:
I would leave `.top` and `.banner` classes in `_general.scss`. It was my intention for this file to gather all general styling and group of classes that are not exactly modular in BEM sense. Maybe for this group of classes (all not BEMish and not _general) we can have a separate file, like `_auxiliary.scss` or alike. 
We could also just rename this file (from `_general.scss` to `_auxiliary.scss`), and keep it all there together. 
I'd try to avoid creating files for each separate class if possible...
What do you think?

Reply:
I was not 100% sure about it. It was just proposition. I am ok with that solution :)
I prefer `_general.scss` over `_auxiliary.scss` (it does not feel natural for me)